tests/tracking/test_tracking.py

```diff
@@ -867,56 +867,58 @@ def test_search_runs():
         mlflow.log_param("p2", "aa")
         mlflow.set_tag("t2", "second-tag-val")
 
-    def verify_runs(runs, expected_set):
-        assert set([r.info.run_id for r in runs]) == set([logged_runs[r] for r in expected_set])
-
     experiment_id = MlflowClient().get_experiment_by_name("exp-for-search").experiment_id
 
     # 2 runs in this experiment
     assert len(MlflowClient().list_run_infos(experiment_id, ViewType.ACTIVE_ONLY)) == 2
 
+    def run_and_verify(experiment_ids, query, check, **kwargs):
+        runs = MlflowClient().search_runs(experiment_ids, query, **kwargs)
+
+        assert set([r.info.run_id for r in runs]) == set([logged_runs[r] for r in check])
+
+        # verify as_pandas=False returns the same as mlflow_client.search_runs
+        fluent_search_runs_run_ids = [
+            run.info.run_id
+            for run in mlflow.search_runs(experiment_ids, query, as_pandas=False, **kwargs)
+        ]
+        assert fluent_search_runs_run_ids == [run.info.run_id for run in runs]
```

Comment:
I think fluent search runs behavior is best tested in https://github.com/mlflow/mlflow/blob/master/tests/tracking/fluent/test_fluent.py. Can we add a test case there for the new list output format of `search_runs` and add `tests/tracking/fluent/test_fluent.py` to https://github.com/mlflow/mlflow/blob/master/dev/run-python-skinny-tests.sh, skipping any test cases that we don't expect to pass on the skinny client (e.g., the tests for `search_runs` with *pandas* output format)?

Reply:
I added tests but avoided the ones with NaN and datetime that wasn't now

Reply:
Which was 1 test